src/ui/common/DraggableDialog.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Draggable from 'react-draggable';
+import { ModalDialog } from 'react-bootstrap'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
why are we using react-bootstrap and not reacstrap?
why is the dependency not installed explicitly?